src/main/resources/swagger.api/edge-dcb.yaml

```diff
@@ -1,123 +1,138 @@
 openapi: 3.0.0
 info:
   title: Edge DCB API
-  version: v1
+  version: 0.0.1
 servers:
   - url: /dcbService/
 paths:
-  /Transactions/{dcbTransactionId}/status:
-    parameters:
-      - in: path
-        name: dcbTransactionId
-        description: The ReShare DCB ID for the transaction being brokered.
-        required: true
-        schema:
-          type: string
-      - $ref: "#/components/parameters/x-okapi-token"
-      - $ref: "#/components/parameters/x-okapi-tenant"
+  /transactions/{dcbTransactionId}:
+    post:
+      summary: Create circulation request
+      operationId: createCirculationRequest
+      parameters:
+        - $ref: '#/components/parameters/dcbTransactionId'
+      tags:
+        - circulation
+      responses:
+        '201':
+          $ref: '#/components/responses/TransactionStatus'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '409':
+          $ref: '#/components/responses/Conflict'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+      requestBody:
+        $ref: "#/components/requestBodies/DCBObject"
+  /transactions/{dcbTransactionId}/status:
     get:
+      summary: Get transaction status across circulation institutions
+      operationId: getTransactionStatus
+      parameters:
+        - $ref: '#/components/parameters/dcbTransactionId'
+      tags:
+        - circulation
+      responses:
+        '200':
+          $ref: '#/components/responses/TransactionStatusResponse'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
```

Comment:
@kalibek-epam  and @steveellis  
Should we discuss the returned error response codes or messages list with K-int too ?